TechAndCheck/hypatia
# Add stem cells variables

This adds a new env variable `DIFFERENTIATE_AS` to determine which type
of scraper gets run when this is being setup.

To test:
1. Set up DIFFERENTIATE_AS as "instagram".
2. Make sure that both Instagram and Facebook environment variables are set (so that both paths are tested, in production only one needs to be set)
2. `rails t`
